pkg/kubewarden/edit/policies.kubewarden.io.policyserver.vue

```diff
@@ -82,7 +82,8 @@ export default {
       }
 
       try {
-        await this.save(event);
+        // await this.save(event);
```

Comment:
Policy server can't be created, maybe because of this?

Reply:
Oops, forgot to remove that 🙂